RFCs/001-removing-sycl.rst

```diff
@@ -0,0 +1,94 @@
+==========================
+001: Removing SYCL Chapter
+==========================
+
+Remove the SYCL chapter from the oneAPI specification.
+
+See also:
+
+* `WG discussion`_
+
+.. _`WG discussion`: https://github.com/uxlfoundation/spec-working-group/blob/main/meetings/notes.rst#notes
+
+Background
+==========
+
+The SYCL chapter no longer serves a useful purpose and is causing some confusion
+about UXL's role in SYCL. The original motivation for the chapter was that DPC++
+was considered part of oneapi specification, and the chapter specifies the API
+that DPC++ supports, which is SYCL 2020 + extensions. With the adoption of SYCL
+2020, the need for extensions is reduced. DPC++ is not part of UXL. The hope is
+that the SYCL support in DPC++ will be upstreamed in LLVM, and that would be the
+home for specification of extensions, not the oneAPI specification.
+
+After the initial WG discussion, I followed up with Alexey & Greg.
+
+There are several issues:
+
+1. Policy/Documentation about use of SYCL vendor extensions in interfaces that
+   are part of oneapi spec
+2. Policy/Documentation about use of SYCL vendor extensions in UXL open source
+   projects.
+3. Documentation of use of ``std::`` from SYCL kernels
+
+Use of SYCL vendor extensions in interfaces
+===========================================
+
+I searched the header files by cloning all the UXL projects and the sources of
+the spec::
+
+    grep -n -r --include=\*.{h,hpp,rst} -E 'sycl::ext|ext_intel|ext_oneapi' .
```

Comment:
Just a very minor nit.  The SYCL naming convention would not allow namespaces like `sycl::ext_intel` or `sycl::ext_oneapi`.  The correct spelling is `syc;::ext::intel` and `sycl::ext::oneapi`.  However, I think your grep would still have found have found uses of these namespaces because it searched for `sycl::ext`.

Reply:
The RE was covering both types of extensions:
* https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#_extension_namespace
* https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#_names_for_extensions_to_existing_classes_or_enumerations